rpc/handlers_test.go

```diff
@@ -99,3 +100,30 @@ func TestThrottledVMError(t *testing.T) {
 		assert.Equal(t, throttledErr, rpcErr.Data)
 	})
 }
+
+func TestJunoGetNodesFromRoot(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	t.Cleanup(mockCtrl.Finish)
+	mockReader := mocks.NewMockReader(mockCtrl)
+	mockReader.EXPECT().Network().Return(&utils.Mainnet).AnyTimes()
+
+	log := utils.NewNopZapLogger()
+	mockState := mocks.NewMockStateHistoryReader(mockCtrl)
+
+	handler := rpc.New(mockReader, nil, nil, "", utils.Ptr(utils.Mainnet), log)
+
+	t.Run("Key DNE", func(t *testing.T) {
+		mockReader.EXPECT().HeadState().Return(mockState, nopCloser, nil)
+		nodeStorage, err := handler.JunoGetNodesFromRoot(felt.Zero)
+		require.Nil(t, nodeStorage)
+		require.NotEqual(t, err, nil)
+	})
+
+	t.Run("Key Exist", func(t *testing.T) {
+		element := fp1.NewElement(1)
```

Comment:
new(felt.Felt).SetUint64(1)

Reply:
Hi @rianhughes ....
Is there any issue in the way I defined it?? 

```go
element := fp1.NewElement(1)
mockReader.EXPECT().HeadState().Return(mockState, nopCloser, nil)
nodeStorage, err := handler.JunoGetNodesFromRoot(*felt.NewFelt(&element))
```

Like its compiling so I thought why to replace it with `new(felt.Felt).SetUint64(1)`

Reply:
Resolved

Reply:
It's just neater (and safer?) to use the juno felt package, since it's being used everywhere else in the code.